PED/SetPedHeadOverlayColor.md

```diff
@@ -20,7 +20,7 @@ You may need to call [`SetPedHeadBlendData`](#_0x9414E18B9434C2FE) prior to call
 ## Parameters
 * **ped**: The ped entity
 * **overlayID**: An integer representing the overlay id
-* **colorType**: 1 for eyebrows, beards, and chest hair; 2 for blush and lipstick; and 0 otherwise, though not called in those cases. 
+* **colorType**: 1 for eyebrows, beards,makeup, and chest hair; 2 for blush and lipstick; and 0 otherwise, though not called in those cases. 
```

Comment:
You forgot the space after the `,`

Would be

> 1 for eyebrows, beards, makeup, and chest hair; 2 for blush and lipstick; and 0 otherwise, although not called in those cases.

For a better grammar

Reply:
i agree :)